src/services/datasources/hooks/createdBy.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * adds the createdBy field to the data.
+ */
+module.exports = (context) => {
+	context.data.createdBy = (context.params.account || {}).userId;
```

Comment:
For future reference: this will change with #803

Reply:
inwiefern?